src/styles/blocks/container.scss

```
@@ -0,0 +1,12 @@
+@mixin container {
+  margin: 0 auto;
+  max-width: 1110px;
+
+  @media screen and (max-width: 689px) {
+    max-width: 689px;
+  }
+
+  @media screen and (max-width: 327px) {
+    max-width: 327px;
+  }
+}
```

```suggestion
.container {
  margin: 0 auto;
  max-width: 1110px;

  @media screen and (max-width: 689px) {
    max-width: 689px;
  }

  @media screen and (max-width: 327px) {
    max-width: 327px;
  }
}
.button {
  color: yellow;

  @include onTablet {
    color: blue;
  }

  @include onMobile {
    color: yellow;
  }
}

@mixin onTablet {
  @media (max-width: 768px) {
    @content;
  }
}

@mixin onMobile {
  @media (max-width: 360px) {
    @content;
  }
}

```

Вынеси потом эти миксины в `src/styles/utils/_media.scss`